credential-registry/client/tob-web/src/app/topic/form.component.ts

```diff
@@ -129,7 +129,7 @@ export class TopicFormComponent implements OnInit, OnDestroy {
         if(this._loader.result.error)
           state = true;
         else
-          state = this.isLoaded('topic', 'related_to', 'related_from' /*, 'creds'*/);
+          state = this.isLoaded('topic', /*'related_to', */ 'related_to_relations' /*, 'related_from', 'creds'*/);
```

Comment:
Should we have the commented block as a single piece at the end, for readability?